tests/e2e/localImage.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { GolemNetwork, GvmiServer, serveLocalGvmi } from "../../src/experimental";
+const path = require("path");
+
+const localImagePath = path.join(__dirname, "../mock/fixtures/alpine.gvmi");
+
+describe("Local Image", () => {
+  let server: GvmiServer;
+  let golemClient: GolemNetwork;
+  afterAll(async () => {
+    await golemClient.close();
+    await server.close();
+  });
+
+  it("allows the provider to download the image directly from me", async () => {
+    golemClient = new GolemNetwork({});
+    server = serveLocalGvmi(localImagePath);
+    await server.serve();
+    await golemClient.init();
+
+    const job = golemClient.createJob<string>({
+      package: {
+        localImageServer: server,
```

Comment:
Let me put this in context of TaskExecutor for a moment - I think the target picture looks like this:

```ts
const te = await TaskExecutor.create({
  package: "gftp-serve://./path/to/image.gvmi"
});
```

At the end of the day we don't want them to import the GvmiServer anywhere to use it.

Reply:
How will the user get this url? How will they get the image hash? Or do you think the TE should start it's own GVMI server inside and close it once it's finished?